### PR TITLE
LPS-189710: Handless borderless Cancel button when API Application turn into two actions

### PR DESCRIPTION
**Pull request by sergiojimcos**
**What is new?**
- Add a handless function just to know when the `Cancel` button must be borderless or not.

**BEFORE PR**
image

**AFTER PR**
image

**JIRA Related Ticket**
https://liferay.atlassian.net/browse/LPS-189710

### CODE REVIEW

**Comment by sergiojimcos** [comment]
ci:test:sf

File: modules/apps/headless/headless-builder/headless-builder-web/src/main/resources/META-INF/resources/js/components/APIApplicationManagementToolbar.tsx [hunk 1]
```diff
@@ -87,7 +95,7 @@ export function APIApplicationManagementToolbar({
 				<ClayManagementToolbar.ItemList>
 					<ClayButton.Group key={1} spaced>
 						<ClayButton
-							borderless={true}
+							borderless={handleBorderless()}
```
Comment (cgmonte) [inline modules/apps/headless/headless-builder/headless-builder-web/src/main/resources/META-INF/resources/js/components/APIApplicationManagementToolbar.tsx:98]: I think we should just delete the `borderless` prop and have the button always with a border. Why should the button look different depending on publish status?

**Review by cgmonte — COMMENTED** [review]
Hi @sergiojimcos, thanks for fixing this! I used the borderless prop beucause the cancel button in design mockups was borderless, but I do agree that it looks better with a border. Please, see my comment, as I think that there should be no condition to show or not the border. It should always have a border. No? Let me know if you have other information.

File: modules/apps/headless/headless-builder/headless-builder-web/src/main/resources/META-INF/resources/js/components/APIApplicationManagementToolbar.tsx (hunk 1 above)
Comment (cgmonte) [reply modules/apps/headless/headless-builder/headless-builder-web/src/main/resources/META-INF/resources/js/components/APIApplicationManagementToolbar.tsx:98]: Hey, after talking with the team, as you probably saw, your solution is right! Cancel has no border if it is the third button :clap:

**But!** I see `handle` as a sufix added to function names when the function handles some event. In this case there is no event, and what the function does is not directly related to the border, it just checks if the API its unpublished. So a proper name to this function could be `isUnpublished`.

**But!** `itemData.applicationStatus?.key === 'unpublished'` already returns the boolean we want. We could just store it in `isUnpublished`, like this `const isUnpublished = itemData.applicationStatus?.key === 'unpublished'`.

**But!** It is a short expression, used only once, so we should just use inline `borderless={itemData.applicationStatus?.key === 'unpublished'}`

Comment (sergiojimcos) [reply modules/apps/headless/headless-builder/headless-builder-web/src/main/resources/META-INF/resources/js/components/APIApplicationManagementToolbar.tsx:98]: Great!! @cgmonte Thanks for your review, make sense a lot

**Comment by sergiojimcos** [comment]
ci:test:sf

**Comment by sergiojimcos** [comment]
ci:test:relevant

**Comment by sergiojimcos** [comment]
ci:test:relevant

**Review by cgmonte — APPROVED** [review]
LGTM!

**Comment by cgmonte** [comment]
co:forward

**Comment by yanan** [comment]
Test failures unrelated, ok to forward

**Comment by sergiojimcos** [comment]
Sent to Brian: https://github.com/brianchandotcom/liferay-portal/pull/138051

**Pull request closed without merging**